src/main/java/com/revature/services/EvalLogicImpl.java

```diff
@@ -17,34 +15,67 @@
 
 	@Autowired
 	private EvalRepository dao;
+	
+
+//CREATE----------------------------
 
+//RETRIEVE--------------------------
 	@Override
 	public Eval getEvalById(int id) {
-		return dao.findOne(id);
-
+		Eval eval = dao.findOne(id);
+		return eval;
 	}
 	
 	@Override
-	public List<Eval> getEvalsByBatch(int id){
-		
-		return dao.findAllByBatchId(id);
+	public Page<Eval> getEvalsByBatch(Pageable pageable, int id, String evalParam){
+		Page<Eval> evals;
+		if (evalParam.equalsIgnoreCase("both")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Move the "both" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1132)